tests/acceptance/guid-node/forks-test.ts

```diff
@@ -0,0 +1,146 @@
+import { percySnapshot } from 'ember-percy';
+import { module, test } from 'qunit';
+
+import { forkNode } from 'ember-osf-web/mirage/helpers';
+import { click, currentURL, setupOSFApplicationTest, visit } from 'ember-osf-web/tests/helpers';
+
+import Node from 'ember-osf-web/models/node';
+import { Permission } from 'ember-osf-web/models/osf-model';
+
+module('Acceptance | guid-node/forks', hooks => {
+    setupOSFApplicationTest(hooks);
+    setupMirage(hooks);
+
+    test('logged out, no forks', async assert => {
+        const node = server.create<Node>('node', { id: 'f0rk5', currentUserPermissions: [] });
+        const url = `/${node.id}/forks`;
+
+        await visit(url);
+        assert.equal(currentURL(), url, `We are on ${url}`);
+        assert.equal(currentRouteName(), 'guid-node.forks', 'We are at guid-node.forks');
+        await percySnapshot(assert);
+        assert.dom('[data-test-new-fork-button]').doesNotExist();
+        assert.dom('[data-test-forks-info]')
+            .hasText('Forks you have permission to view are shown here.');
+    });
+
+    test('logged out, 1 fork', async assert => {
+        const title = 'Test Title';
+        const node = server.create('node', { id: 'f0rk5', title });
+
+        const fork = forkNode(
+            server,
+            node as ModelInstance<Node>,
+            {
+                currentUserPermissions: Object.values(Permission),
+            },
+        );
+        const contributorUser = server.create('user');
+        server.create('contributor', { node: fork, users: contributorUser });
+
+        const url = `/${node.id}/forks`;
+        await visit(url);
+        assert.equal(currentURL(), url, `We are on ${url}`);
+        assert.dom('[data-test-new-fork-button]').doesNotExist();
+        assert.dom('[data-test-node-card]').exists({ count: 1 });
+        assert.dom('[data-test-node-card-heading]').includesText(title);
+    });
+
+    test('logged in admin, no forks', async assert => {
+        server.create('user', 'loggedIn');
+        const node = server.create<Node>('node', { id: 'f0rk5', currentUserPermissions: [Permission.Admin] });
+        const url = `/${node.id}/forks`;
+
+        await visit(url);
+        assert.equal(currentURL(), url, `We are on ${url}`);
+        assert.dom('[data-test-new-fork-button]').exists({ count: 1 });
+        assert.dom('[data-test-forks-info]')
+            .hasText('Forks you have permission to view are shown here.');
+    });
+
+    test('logged in admin, 1 fork', async assert => {
+        const contributorUser = server.create('user', 'loggedIn');
+        const node = server.create<Node>('node', {
+            id: 'decaf',
+            title: 'Test Title',
+            currentUserPermissions: [Permission.Admin],
+        });
+        server.create('contributor', { node, users: contributorUser });
+        const fork = forkNode(
+            server,
+            node as ModelInstance<Node>,
```

Comment:
should just be able to do:
```suggestion
            node,
```
because `server.create<Node>`